### PR TITLE
chore: update formatting for READMEs

### PR DESCRIPTION
**Pull request by cyber-eternal**
- Update formatting in graph-feature-service/README.md
- Update formatting in science/search/ingester/config/README.md
- Update formatting in timelineranker/README.md
- Update formatting in visibilitylib/README.md

### CODE REVIEW

File: graph-feature-service/README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Graph Feature Service
 
-Graph Feature Service (GFS) is a distributed system that can provide various graph features for given pairs of users. For instance, given source user A and candidate user C, GFS can answer questions like “how many of A’s followings have favorited C”, “how many of A’s followings are following C”, and “how much C is similar to the users that A has favorited“.
\ No newline at end of file
+Graph Feature Service (GFS) is a distributed system that can provide various graph features for given pairs of users. For instance, given source user A and candidate user C, GFS can answer questions like “how many of A’s followings have favorited C”, “how many of A’s followings are following C”, and “how much C is similar to the users that A has favorited“.
```
Comment (kapilepatel) [inline graph-feature-service/README.md:3]: What change is there?

File: product-mixer/README.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
-Product Mixer
-=============
+# Product Mixer
```
Comment (kapilepatel) [inline product-mixer/README.md:1]: this looks good

File: science/search/ingester/config/README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
 ## Ingester Configs
-This directory contains pipeline configurations for the tweet ingesters (realtime, protected and realtime_cg) and the user-updates ingester. The pipeline configurations define an ordered sequence of stages that the tweet or user update goes through before reaching Earlybird. Source code for the various stages referenced in the configs can be found at src/java/com/twitter/search/ingester/pipeline/twitter.
\ No newline at end of file
+
```
Comment (kapilepatel) [inline science/search/ingester/config/README.md:2]: looks good, a new line in md file, but no change in UI visibility.

File: simclusters-ann/README.md [hunk 4]
```diff
@@ -5,50 +5,50 @@ SimClusters ANN is a service that returns tweet candidate recommendations given
...
 
 ## Background
-SimClusters V2 runtime infra introduces the SimClusters and its online and offline approaches. A heron job builds the mapping between SimClusters and Tweets. The job saves top 400 Tweets for a SimClusters and top 100 SimClusters for a Tweet. Favorite score and follow score are two types of tweet score.  In the document, the top 100 SimClusters based on the favorite score for a Tweet stands for the Tweet SimClusters Embedding. 
 
-The cosine similarity between two Tweet SimClusters Embedding presents the relevant level of two tweets in SimCluster space. The score varies from 0 to 1. The high cosine similarity score(>= 0.7 in Prod) means that the users who like two tweets share the same SimClusters. 
+SimClusters V2 runtime infra introduces the SimClusters and its online and offline approaches. A heron job builds the mapping between SimClusters and Tweets. The job saves top 400 Tweets for a SimClusters and top 100 SimClusters for a Tweet. Favorite score and follow score are two types of tweet score. In the document, the top 100 SimClusters based on the favorite score for a Tweet stands for the Tweet SimClusters Embedding.
```
Comment (kapilepatel) [inline simclusters-ann/README.md:9]: Added a new line in MD, looks good

File: simclusters-ann/README.md [hunk 5]
```diff
@@ -5,50 +5,50 @@ SimClusters ANN is a service that returns tweet candidate recommendations given
...
-
-2. Fetch top *M* tweets for each Top *N* SimClusters based on SV. In Prod, *M = 400*, *N = 50*.  Tweets may appear in multiple SimClusters. 
- 
-|   |   |   |   |
-|---|---|---|---|
-| SC1  | T1:Score  | T2: Score  | ...   |
-| SC2 |  T3: Score | T4: Score  |  ... |
+1. Provide a source SimCluster Embedding _SV_, _SV = [(SC1, Score), (SC2, Score), (SC3, Score) …]_
 
+2. Fetch top _M_ tweets for each Top _N_ SimClusters based on SV. In Prod, _M = 400_, _N = 50_. Tweets may appear in multiple SimClusters.
```
Comment (kapilepatel) [inline simclusters-ann/README.md:21]: Should we change SV to _SV_?

File: simclusters-ann/README.md [hunk 6]
```diff
@@ -5,50 +5,50 @@ SimClusters ANN is a service that returns tweet candidate recommendations given
...
 
-3. Based on the previous table, generate an *(M x N) x N* Matrix *R*. The *R* represents the approximate SimCluster embeddings for *MxN* tweets. The embedding only contains top *N* SimClusters from *SV*. Only top *M* tweets from each SimCluster have the score. Others are 0. 
+|     |           |           |     |
```
Comment (kapilepatel) [inline simclusters-ann/README.md:23]: What is this table with empty header, shall we remove empty header in original file?

File: simclusters-ann/README.md [hunk 7]
```diff
@@ -5,50 +5,50 @@ SimClusters ANN is a service that returns tweet candidate recommendations given
...
+5. Fetch _X_ tweet SimClusters Embedding, Calculate Cosine Similarity between _X_ tweets and _SV_, Return top _Y_ above a certain threshold _Z_.
 
-The idea of the approximate algorithm is based on the assumption that the higher dot-product between source tweets’ SimCluster embedding and candidate tweet’s limited SimCluster Embedding, the possibility that these two tweets are relevant is higher. Additional Cosine Similarity filter is to guarantee that the results are not affected by popularity bias.  
+Approximate Cosine Similarity is an approximate algorithm. Instead of fetching _M _ N* tweets embedding, it only fetches *X* tweets embedding. In prod, *X / M _ N _ 100% = 6%\*. Based on the metrics during TWISTLY development, most of the response time is consumed by I/O operation. The Approximate Cosine Similarity is a good approach to save a large amount of response time.
```
Comment (kapilepatel) [inline simclusters-ann/README.md:40]: Please fix this from _M _ N* to _M _ * _N_

Comment (kapilepatel) [reply simclusters-ann/README.md:40]: also *X* to _X_

Comment (kapilepatel) [reply simclusters-ann/README.md:40]: also fix *X / M _ N _ 100% = 6%\*

**Review by kapilepatel — COMMENTED** [review]
Please see my comments, happy to review again.

File: simclusters-ann/README.md (hunk 5 above)
Comment (cyber-eternal) [reply simclusters-ann/README.md:21]: Not sure. Lets leave it for now :)

File: graph-feature-service/README.md (hunk 1 above)
Comment (cyber-eternal) [reply graph-feature-service/README.md:3]: Just formatting

File: simclusters-ann/README.md (hunk 6 above)
Comment (cyber-eternal) [reply simclusters-ann/README.md:23]: Maybe it would be changed in a scope of future commits

**Pull request closed without merging**